backend/FwLite/MiniLcm/SyncHelpers/SenseSync.cs

```diff
@@ -32,13 +32,10 @@ public static async Task<int> Sync(Guid entryId,
         patchDocument.Operations.AddRange(MultiStringDiff.GetMultiStringDiff<Sense>(nameof(Sense.Definition),
             beforeSense.Definition,
             afterSense.Definition));
-        if (beforeSense.PartOfSpeech != afterSense.PartOfSpeech)
-        {
-            patchDocument.Replace(sense => sense.PartOfSpeech, afterSense.PartOfSpeech);
-        }
 
         if (beforeSense.PartOfSpeechId != afterSense.PartOfSpeechId)
         {
+            patchDocument.Replace(sense => sense.PartOfSpeech, afterSense.PartOfSpeech);
```

Comment:
this will actually get ignored both by flex and crdt, so it can just be removed. The only replace left should be the one for `PartOfSpeechId`